understanding/pointer-gestures.html

```diff
@@ -55,284 +53,179 @@
+               <p><strong>軌跡ベースのジェスチャ</strong>は、終点だけではないインタラクションを含むものである。中間点 (通常はジェスチャの開始近く) を通過することもその意味に影響を与える場合、それは軌跡ベースのジェスチャである。利用者は、ポインタ (始点) を使用し、ポインタ (終点) を解除する前に、少なくとも 1 つの中間点を通過する動きを実行する。中間点は、たとえ完全な軌跡が定義されていないとしても、特定の軌跡を必要とするものとしてジェスチャを定義する。
                </p>
                <figure id="figure-path-based-gesture-2">
-                  				<img src="img/path-based-gesture-2.png" width="400" alt="Hand showing a starting touch, 1. Moving through a second point, 2. Going to one of several points,3." />
+                  				<img src="img/path-based-gesture-2.png" width="400" alt="1、タッチの開始を示す手。2、2 つ目の点を通過する。3、複数の点の 1 つに行く。" />
                   				
-                  <figcaption>Figure 2 A <strong>path-based gesture</strong> involves starting a pointer movement that goes through at least one intermediate
-                     point before the end-point. The end-point may be a continuation, or allow for various
-                     movements.
+                  <figcaption>図 2 <strong>軌跡ベースのジェスチャ</strong>では、終点の前に少なくとも 1 つの中間点を通過するポインタの動きを開始する。終点は継続するものであってもよく、又はさまざまな動きを可能にするものであってもよい。
                   </figcaption>
                   			
                </figure>
-               <p>Examples of path-based gestures include swiping, sliders and carousels dependent on
-                  the direction of interaction, and other gestures which trace a prescribed path such
-                  as drawing a specific shape. Such paths may be drawn with a finger or stylus on a
-                  touchscreen, graphics tablet, or trackpad, or with a mouse, joystick, or similar pointer
-                  device.
+               <p>軌跡ベースのジェスチャの例には、インタラクションの方向に応じたスワイプ、スライダー、カルーセル、及び特定の形状の描画などの指定された軌跡をトレースするその他のジェスチャが含まれる。このような軌跡は、タッチスクリーン、ペンタブレットもしくはトラックパッド上で指もしくはスタイラスペンを用いて、又はマウス、ジョイスティックもしくは同様のポインタデバイスを用いて描画されるかもしれない。
                </p>
-               <p>Dragging is a movement where the user picks up an object with a pointer (such as mouse
-                  cursor or a finger) and moves it to some other position. This movement from start
-                  point to end point does not require the user to follow any particular path or direction.
-                  Dragging is therefore not path-based. In contrast, a path-based pointer gesture requires
-                  the traversal of an intermediate point, which is a technical way of expressing that
-                  the directionality and possibly speed of the gesture communicates a particular command
-                  to the system. Dragging motions are covered in <a href="https://www.w3.org/WAI/WCAG22/Understanding/dragging.html">Success Criterion 2.5.7: Dragging</a>.
+               <p>ドラッグとは、利用者がポインタ (マウスカーソル、指など) でオブジェクトをつかんで、他の位置に移動させる動作のことである。この始点から終点への動作は、利用者に特定の軌跡又は方向をたどることを要求しない。したがって、ドラッグは軌跡ベースではない。これに対して、軌跡ベースのポインタジェスチャでは中間点の通過が求められるが、これは、ジェスチャの方向性、及びもしかしたら速度が、システムに特定のコマンドを伝えることを表現する技術的な方法である。ドラッグ動作については<a href="dragging.html">達成基準 2.5.7: ドラッグ動作</a>で扱う。
```

Comment:
「もしかしたら」はちょっと口語的すぎるように思いました。「場合によっては」というくらいでどうでしょうか。

```suggestion
               <p>ドラッグとは、利用者がポインタ (マウスカーソル、指など) でオブジェクトをつかんで、他の位置に移動させる動作のことである。この始点から終点への動作は、利用者に特定の軌跡又は方向をたどることを要求しない。したがって、ドラッグは軌跡ベースではない。これに対して、軌跡ベースのポインタジェスチャでは中間点の通過が求められるが、これは、ジェスチャの方向性、及び場合によっては速度が、システムに特定のコマンドを伝えることを表現する技術的な方法である。ドラッグ動作については<a href="dragging.html">達成基準 2.5.7: ドラッグ動作</a>で扱う。
```

Reply:
そのように修正しました。